js/interface_common.js

```diff
@@ -544,18 +549,49 @@ var replyInitPopup = function(e, post)
 }
 
 //abre o menu dropdown de configurações
-var dropDownMenu = function( e )
-{
-    var $configMenu = $( ".config-menu" );
-    $configMenu.slideToggle( "fast" );
-    e.stopPropagation();
+function dropDownMenu() {
+    $( ".config-menu" ).slideToggle( "fast" );
 }
 
 //fecha o config menu ao clicar em qualquer lugar da tela
-var closeThis = function()
-{
+function closeThis() {
     $( this ).slideUp( "fast" );
-};
+}
+
+function toggleFollowButton(button, followingUser) {
+    if (!button || !followingUser)
+        return;
+
+    button
+        .removeClass("follow")
+        .addClass("unfollow")
+        .unbind("click")
+        .bind("click",
+            (function(e) {
+                unfollow(this.toString(),
+                    (function() {
+                        this
+                            .removeClass("unfollow")
+                            .addClass("follow")
+                            .unbind("click")
+                            .bind("click", userClickFollow);
+
+                        if ($.Options.getTheme() === 'nin') {
```

Comment:
I'd like to avoid adding any theme-specific javascript code here.
Are you sure we can't have this nin-specific behavior implemented from theme_nin/js/theme_option.js file which is automatically included?